Common/src/main/java/com/blamejared/crafttweaker/api/zencode/CraftTweakerGlobals.java

```diff
@@ -4,17 +4,23 @@
 import com.blamejared.crafttweaker.api.CraftTweakerAPI;
 import com.blamejared.crafttweaker.api.CraftTweakerConstants;
 import com.blamejared.crafttweaker.api.annotation.ZenRegister;
+import com.google.common.base.Suppliers;
+import org.apache.logging.log4j.Logger;
 import org.openzen.zencode.java.ZenCodeGlobals;
 import org.openzen.zencode.java.ZenCodeType;
 
+import java.util.function.Supplier;
+
 @ZenRegister(loaders = CraftTweakerConstants.ALL_LOADERS_MARKER)
 @ZenCodeType.Name("crafttweaker.api.Globals")
 public final class CraftTweakerGlobals {
     
+    private static final Supplier<Logger> LOGGER = Suppliers.memoize(() -> CraftTweakerAPI.getLogger("Script"));
```

Comment:
should this not be in CommonLoggers?

Reply:
Are we expecting somebody else to use it?